users/service.go

```diff
@@ -685,10 +697,10 @@ func (svc service) updateClientPolicy(ctx context.Context, userID string, role m
 			Object:      auth.MagistralaObject,
 		})
 		if err != nil {
-			return errors.Wrap(svcerr.ErrDeletePolicies, err)
+			return errors.Wrap(svcerr.ErrUpdateEntity, err)
 		}
 		if !resp.Deleted {
```

Comment:
Please revert back them to original, We can handle errors in ticket https://github.com/absmach/magistrala/issues/2202